### PR TITLE
Translate

### PR DESCRIPTION
**Pull request by panxp**
Translating  3.0/zh/core-libraries/xml.rst to chinese languge.

### CODE REVIEW

File: zh/core-utility-libraries/xml.rst [hunk 1]
```diff
@@ -1,18 +1,184 @@
+
+
+- Xml
+- 创建XML对象
+- 将Xml转换成数组
+- 将数组转换成Xml
+    - 使用命名空间
+    - 创建一个子节点
```
Comment (markstory) [inline zh/core-utility-libraries/xml.rst]: The Table of Contents is automatically generated by the documentation tool.

File: zh/plugins.rst [hunk 2]
```diff
@@ -19,4 +19,62 @@ CakePHP 让你可以建立一个控制器、模型和视图的组合，并且把
...
+
+**Plugin::loadAll() 不能加载在 ``vendor/cakephp-plugins.php`` 中未定义vendor 命名空间的插件。**
+
+
+Autoloading Plugin Classes
+When using bake for creating a plugin or when installing a plugin using Composer, you don’t typically need to make any changes to your application in order to make CakePHP recognize the classes that live inside it.
+
+In any other cases you may need to modify your application’s composer.json file to contain the following information:
+
+
```
Comment (ravage84) [inline zh/plugins.rst]: Do we need so many white lines?

File: zh/core-utility-libraries/xml.rst (hunk 1 above)
Comment (panxp) [inline zh/core-utility-libraries/xml.rst]: ok

File: zh/plugins.rst (hunk 2 above)
Comment (panxp) [inline zh/plugins.rst]: I‘m first work on translation, thank you for reminding.

**Pull request closed without merging**